src/DataBlobs/services/BlobEntry.jl

```diff
@@ -93,7 +92,9 @@ end
 
 #TODO consider changing this to use `first` as in julia's findfirst - as in findfirstBlobEntry->label::Symbol
 # or getBlobEntryFirst, btw this returns a vector in some other places
-function getBlobEntry(var::AbstractDFGVariable, key::Regex)
+@deprecate getBlobEntry(var::AbstractDFGVariable, key::Regex) getfirstBlobEntry(var, key)
+
+function getfirstBlobEntry(var::AbstractDFGVariable, key::Regex)
```

Comment:
```suggestion
function getBlobEntryFirst(var::AbstractDFGVariable, key::Regex)
```

Reply:
Ok, I'll go for that naming, coding decide what way around